src/main/java/bio/terra/workspace/service/resource/controlled/ControlledResourceService.java

```diff
@@ -208,4 +221,23 @@ public String deleteControlledGcsBucket(
 
     return jobBuilder.submit();
   }
+
+  private void validateOnlySelfAssignment(
+      ControlledResource controlledResource, AuthenticatedUserRequest userRequest) {
+    if (!controlledResource.getAccessScope().equals(AccessScopeType.ACCESS_SCOPE_PRIVATE)) {
+      // No need to handle SHARED resources
+      return;
+    }
+      final String requestUserEmail = samService.getEmailFromToken(userRequest.getRequiredToken());
+      // If there is no assigned user, this condition is satisfied.
```

Comment:
A private resource with no assigned user will currently pass this check, but we still rely on this field when creating a controlled resource. If you're marking this field deprecated in this PR, can you also change the place it's used: https://github.com/DataBiosphere/terra-workspace-manager/blob/dev/src/main/java/bio/terra/workspace/service/iam/SamService.java#L545